packages/core/src/components/dropdown/dropdown.tsx

```diff
@@ -72,11 +72,11 @@ export class TdsDropdown {
   @Prop() noResultText?: string = 'No result';
 
   /** Default value selected in the Dropdown. */
-  @Prop() defaultValue: string;
+  @Prop() initialValue: string | number | (string | number)[];
```

Comment:
If we rename prop it is breaking change. Can we keep old one or there is strong need to change it?